SConstruct

```diff
@@ -3025,7 +3041,7 @@ usdEnvAppends = {
 		"-Wno-deprecated" if env["PLATFORM"] != "win32" else "",
 		"-DBUILD_COMPONENT_SRC_PREFIX=",
 		"-DBUILD_OPTLEVEL_DEV",
-	],
+	] + formatSystemIncludes( usdEnv, "$USD_INCLUDE_PATH" ),
```

Comment:
We'll also need `$PYTHON_INCLUDE_PATH` included here.

Reply:
Great, included in d5287b7

Reply:
LGTM, thanks :)